frontend/services/entities/mdm.ts

```diff
@@ -4,6 +4,12 @@ import sendRequest from "services";
 import endpoints from "utilities/endpoints";
 import { buildQueryStringFromParams } from "utilities/url";
 
+export interface IEulaMetadataResponse {
```

Comment:
I like this explicit callout that this is the response. Definitely a pattern we should backtrack and add to all our response data.